stbt_rig.py

```diff
@@ -909,6 +924,117 @@ def push_git_snapshot(self, branch):
         return commit_sha
 
 
+try:
+    import pytest
+
+    def pytest_addoption(parser):
+        group = parser.getgroup("stbt", "stb-tester REST API")
+        for arg in itertools.chain(ARGS, RUN_ARGS):
+            if arg.cmdline_only:
+                continue
+            d = {k: v for k, v in arg._asdict().items()
+                 if (k not in ['name', 'completer', 'cmdline_only'] and
+                     v is not None)}
+            group.addoption(arg.longname, **d)
+
+
+    def pytest_collect_file(path, parent):
+        if path.ext == ".py":
+            return StbtCollector(path, parent)
+        else:
+            return None
+
+
+    class StbtCollector(pytest.File):
+        def collect(self):
+            with open(self.name) as f:
+                # We implement our own parsing to avoid import stbt ImportErrors
+                for line in f:
+                    m = re.match(r'def (test_.*)\s*\(\):\s*$', line)
```

Comment:
This should be `r'^def\s\s*test_[a-zA-Z0-9_]*'`

Reply:
(Plus the necessary parentheses to get `group(1)`).

Reply:
Done.